2018/asee2018.Rmd

```diff
@@ -660,6 +663,52 @@ ggplot(data_change.in.confidence_tally,
 ggsave("figures/change_in_confidence.png")
 ```
 
+__Breakdown by Status__
+
+```{r}
+status <- unique(round1data$Position)
+status <- factor(status[!is.na(status)])
+```
+
+What is your position?
+
+```{r}
+status <- factor(status, levels(status)[c(7, 3, 6, 2, 1, 5, 4)])
+# Display ordering.
+print(levels(status))
+
+round1data$Position <-
+  factor(round1data$Position, levels = levels(status))
+
+change_in_confidence_by_status <- round1data %>%
+  group_by(`Change-In-Confidence`, Position) %>%
+  tally() %>%
+  filter(!is.na(`Change-In-Confidence`)) %>%
+  group_by(Position) %>%
+  mutate(n_status = sum(n)) %>%
+  filter(!is.na(Position)) %>%
+  mutate(perc = n / n_status * 100)
+
+ggplot(data = change_in_confidence_by_status,
+       aes(x = `Change-In-Confidence`, y = Position)) +
+  geom_tile(aes(fill = perc), colour = "white") +
+  geom_text(aes(label = paste0(round(perc, 1), "%"))) +
```

Comment:
@kariljordan We may want to display more information by showing absolute numbers here (instead of the redundant percentage). Feel free to replace this line by `geom_text(aes(label = n)) +`.